Autopilot/Test/Src/DataStructures/Test_Vector_Class.cpp

```diff
@@ -0,0 +1,109 @@
+#include <gtest/gtest.h>
+#include "math.h"
```

Comment:
same thing about cmath.

tbh, I'm surprised cmake found this. "" is used for user libraries while <> is used for system libraries.

Reply:
Fixed!